docs/Permissions/Permissioning.md

```diff
@@ -95,7 +95,7 @@ can synchronise and add blocks containing transactions from accounts that are no
     Node 2 now has a transaction in the blockchain from Account A which is not on the accounts whitelist for Node 2.   
 
 !!! note
-    Each node has a [`permissions_config.toml`](#permissions-configuration-file) file which means nodes in the network can have different accounts whitelists. 
+    Each node has a [permissions configuration file](#permissions-configuration-file) which means nodes in the network can have different accounts whitelists. 
     This means a transaction can be successfully submitted by Node A from an account in the Node A whitelist but rejected by 
     Node B to which it is propagated if the account is not in the Node B whitelist. 
```

Comment:
Maybe add something like:

`..the account is not in the Node B whitelist. This will cause the nodes to never reach consensus and can cause a chain fork`.

Reply:
> Maybe add something like:
> 
> `..the account is not in the Node B whitelist. This will cause the nodes to never reach consensus and can cause a chain fork`.

I thought we dont cause forks at the moment? Once a transaction is included in a block it will be accepted regardless if the account is in the whitelist or not. Support for this will be included in on-chain permissioning.

Reply:
Arash, you are right. We won't cause a fork if the transaction has been mined in a different node.

Reply:
We need then to add a note about how we won't validate the account whitelist when importing and validating blocks. :)